### PR TITLE
Update cluster config- changed cvg_type to 'ios'.

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
# Problem Statement
- Change cvg_type to 'ios' in cluster.yaml.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: conf/cluster.yaml.sample [hunk 1]
```diff
@@ -18,7 +18,7 @@ cluster:
             - message_bus
       storage:
         - name: cvg-01
-          type: cas       # io (default), cas
+          type: ios       # io (default), cas
```
Comment (cdeshmukh) [inline conf/cluster.yaml.sample:21]: comment here says 'io', shouldn't it be ios ?

**Review by nitin-seagate — APPROVED** [review]
Same comment as @cdeshmukh

File: conf/cluster.yaml.sample (hunk 1 above)
Comment (shriya-deshmukh) [reply conf/cluster.yaml.sample]: Yes, updated the comment.

**Review by cdeshmukh — APPROVED** [review]

**Review by madhavemuri — APPROVED** [review]

**Pull request merged**